backend/src/graph-info/parseNeo4jEntityInfo.ts

```diff
@@ -0,0 +1,58 @@
+import { Record as Neo4jRecord } from 'neo4j-driver';
+import { EntityType } from '../shared/graph-information/EntityType';
+import { EntityTypeAttribute } from '../shared/graph-information/EntityTypeAttribute';
+
+/**
+ * Converts result from neo4j-query 'CALL db.schema.nodeTypeProperties'
+ * or 'CALL db.schema.relTypeProperties' to generic list of
+ * {@link EntityType}.
+ *
+ * @param result query-result from neo4j
+ * @param type is result about nodes oder edges/rels?
```

Comment:
change "oder" to "or" and say "about rels/edges or nodes" since "rels" is the first option in type